.tito/lib/origin/builder/__init__.py

```diff
@@ -55,15 +55,37 @@ def _setup_test_specfile(self):
             )
             # Custom Openshift v3 stuff follows, everything above is the standard
             # builder
-            cmd = '. ./hack/common.sh ; echo $(os::build::ldflags)'
+            
+            ## Fixup os_git_vars
```

Comment:
Well, actually not the same way. You don't need to do `echo $(funcname)`, you can simply  do `funcname`. But that's a nitpick